### PR TITLE
SPA userinterface

### PR DESCRIPTION
**Pull request by Therealadetunji**
Use React components. ✔️ 
styling done using css and html written with react. ✔️

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
+1. Clone the repository: https://github.com/Therealadetunji/Math-Magicians.git on vscode
+2. run npm i
+3. make sure you have version of eslint in the terminal description.
+4. npm run build
+5. npm start
 
 To get a local copy up and running follow these simple example steps.
```
Comment (Ismailco) [inline README.md]: - [ ] [OPTIONAL] Kindly remove all the empty headers your not using from your README so it look more professional
```
### Prerequisites
### Setup
### Install
### Usage
### Run tests

### Deployment
```

File: src/index.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+// eslint-disable-next-line react/prefer-stateless-function
```
Comment (Ismailco) [inline src/index.js]: - [ ] Kindly fix linters, don't try to skip them, they are here to make your code follow best practices and make you a better programmer

File: src/index.css [hunk 3]
```diff
@@ -1,27 +1,60 @@
...
+  font-size: 1rem;
+  font-weight: 300;
+  text-transform: uppercase;
+  border: 1px solid #d4d4d2;
+  color: #000;
+}
+
+/* .calculatorButtons > button:hover {
+  background-color: #505050;
+} */
```
Comment (Ismailco) [inline src/index.css]: - [ ] For best practices, kindly consider removing all unused commented code from your repo, it makes your code messy

File: src/App.js [hunk 4]
```diff
@@ -1,30 +1,10 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends PureComponent {
```
Comment (Ismailco) [inline src/App.js:5]: - [ ] Your app doesn't run properly, you can check your code to see if everything is installed correctly, I got this error when trying to open the app for preview. kindly consider fixing it

Screenshot_20220810_175447

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @Therealadetunji, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️  No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Therealadetunji, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean/Organized code

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**